### DISCUSSION TITLE
Tem algum repositório com grade de estudo para programação ?

### DISCUSSION THREAD

**Discussion by williamdoxv** [williamWHC/Rocketseat#4] [Q&A] [answered]
Alguma plataforma de estudo para aprender programação com um reço bom ?

**Comment by williamWHC** [comment, marked as answer]
Tem a plataforma Udemy, que tem ótimos cursos com um preço bem acessível, por volta de 27,90

**Comment by williamdoxv** [comment]
Qual a melhor faculdade de tecnologia de São Paulo ?

**Comment by williamWHC** [comment]
Na minha opinião é a FIAP.
